tgui/packages/tgui-modal/index.js

```diff
@@ -15,10 +14,15 @@ const setupApp = () => {
     document.addEventListener('DOMContentLoaded', setupApp);
     return;
   }
-  setupGlobalEvents({
-    ignoreWindowFocus: true,
-  });
-  renderApp();
+  // Enable hot module reloading
+  if (module.hot) {
+    setupHotReloading();
+    module.hot.accept([
+      './components',
+    ], () => {
+      renderApp();
+    });
```

Comment:
```suggestion
    ], renderApp);
```

Right?

Reply:
this is copy pasted directly over from main tgui package. maybe? this syntax is a little wonky

Reply:
If this is going to be inlined into html, then hot reloading is not needed at all.